assets/component-menu-drawer.css

```diff
@@ -223,6 +222,10 @@ details[open].menu-opening > .menu-drawer__submenu {
   margin-right: 1rem;
 }
 
+.menu-drawer__account:hover .icon-account {
+  transform: scale(1.07);
+}
+
```

Comment:
For the opacity of hover on icons, I have updated the social icons, drawer icons, share icon and header icons. I havent updated the close button on the modal or any other icons since they arent ctas. Let me know if that makes sense @wiktoriaswiecicka @Oliviammarcello